vcx/wrappers/node/src/utils/memory-management-helpers.ts

```diff
@@ -1,21 +1,14 @@
 import * as weak from 'weak'
-import { VCXInternalError } from '../errors'
 
 export abstract class GCWatcher {
   protected abstract _releaseFn: any
   // LibVCX handles invalid handles
   private _handleRef!: number
 
-  public async release (): Promise<number> {
-    try {
-      const rc = this._releaseFn(this._handleRef)
-      if (rc) {
-        throw rc
-      }
-      return rc
-    } catch (err) {
-      throw new VCXInternalError(err)
-    }
+  public async release (): Promise<void> {
```

Comment:
Each object extends GCWatcher, so this is where the release function is defined.  It no longer throws an error.